kubernetes/app/api-db.yaml

```diff
@@ -31,6 +31,24 @@ spec:
               name: db-config
```

Comment:
1. In the [configuration document under Dex](https://github.com/SovereignCloudStack/status-page-deployment/blob/main/docs/configuration.md#dex) is described how to configure  Dex. `dex-secrets.env` is the only real secret in the kind environment and therefor is not shared, like all other environments. This might need better mentioning in the docs.
2. Fully agree